rust/src/trans/manipulation/make_is_equal.tex

```diff
@@ -0,0 +1,84 @@
+\documentclass{article}
+\input{../../lib.sty}
+
+\title{\texttt{fn make\_is\_equal}}
+\author{S\'ilvia Casacuberta}
+\date{}
+
+\begin{document}
+
+\maketitle
+
+\contrib
+Proves soundness of \texttt{fn make\_is\_equal} in \asOfCommit{mod.rs}{0db9c6036}.
+The transformation checks if each element in a vector dataset is equivalent to a given value,
+returning a vector of booleans.
+
+\subsection*{Vetting History}
+\begin{itemize}
+    \item \vettingPR{512}
+\end{itemize}
+
+\section{Hoare Triple}
+\subsection*{Precondition}
+To ensure the correctness of the output, we require the following preconditions:
+
+\begin{itemize}
+    \item Type \texttt{TIA} must have trait \texttt{PartialEq}.
+    \item Type \texttt{M} must have trait \texttt{DatasetMetric}.
+    \item \rustdoc{core/trait}{MetricSpace} is implemented for \texttt{(VectorDomain<AtomDomain<TIA>>, M)}. Therefore \texttt{M} is a valid metric on \texttt{VectorDomain<AtomDomain<TIA>>}.
+    \item \rustdoc{core/trait}{MetricSpace} is implemented for \texttt{(VectorDomain<AtomDomain<bool>>, M)}.
```

Comment:
The instances of `>>` are being rendered as guillemets, I think they need to be escaped somehow.

Reply:
It's so gross. Tracked it down to `\usepackage[T1]{fontenc}` in the `lib.sty`. It is [recommended best practice to keep it](https://tex.stackexchange.com/questions/664/why-should-i-use-usepackaget1fontenc). So I inserted extra `{}`.